app/controllers/groups/invites_controller.rb

```diff
@@ -25,17 +25,24 @@ def create
     groups = @group.network? ? recipients.groups : []
     emails = recipients.emails
 
-    reqs = []; mailers = []
+    reqs, mailers = [], []
     unless users.any? or emails.any? or groups.any?
       raise_error('Recipient required')
     end
+
     users.each do |user|
       reqs << RequestToJoinUs.create(created_by: current_user,
         recipient: user, requestable: @group)
     end
+
     groups.each do |group|
-      reqs << RequestToJoinOurNetwork.create(created_by: current_user,
-        recipient: group, requestable: @group)
+      if council = group.council
+        reqs << RequestToJoinOurNetwork.create(created_by: current_user,
+          recipient: council, requestable: @group)
```

Comment:
The idea to only 'ask' the council is good. However if we use the council as the recipient of the request once the request is (approved)[https://github.com/riseuplabs/crabgrass-core/blob/master/app/models/requests/request_to_join_our_network.rb#L36] the council will be added to the network - not the group.

So I think what we want is to notify the people in the council but have the request still deal with the group.